### PR TITLE
Clarify points brought up from public consultation

### PR DESCRIPTION
**Pull request by brettcannon**

### CODE REVIEW

File: pep-0621.rst [hunk 1]
```diff
@@ -270,6 +272,14 @@ Using the data to fill in `core metadata`_ is as follows:
 3. If both ``email`` and ``name`` are provided, the value goes in
    ``Author-email``, with the format ``{name} <{email}>``.
 
+This use of the field is a slight deviation as the core metadata also
+supports a ``Maintainer`` field which going unused by this PEP. The
```
Comment (sersorrel) [inline pep-0621.rst:276]:
```suggestion
supports a ``Maintainer`` field which goes unused by this PEP. The
```

File: pep-0621.rst [hunk 2]
```diff
@@ -93,7 +93,8 @@ Tools MUST specify fields defined by this PEP in a table named
 ``[project]``. No tools may add fields to this table which are not
 defined by this PEP. For tools wishing to store their own settings in
 ``pyproject.toml``, they may use the ``[tool]`` table as defined in
-:pep:`518`.
+:pep:`518`. The lack of a ``[project]`` table implicitly means all
+fields will be dynamically provided by the build tool.
```
Comment (di) [inline pep-0621.rst:97]:
```suggestion
:pep:`518`. The lack of a ``[project]`` table implicitly means the
build tool will dynamically provide all fields.
```

File: pep-0621.rst [hunk 3]
```diff
@@ -270,6 +272,14 @@ Using the data to fill in `core metadata`_ is as follows:
 3. If both ``email`` and ``name`` are provided, the value goes in
    ``Author-email``, with the format ``{name} <{email}>``.
 
+This use of the field is a slight deviation as the core metadata also
+supports a ``Maintainer`` field which goes unused by this PEP. The
+authors of this PEP felt this field was ambiguous and never clearly
+defined. Other ecosystems have selected "author" as the term so use.
```
Comment (sinoroc) [inline pep-0621.rst:278]: "_the term **t**o use_"

**Comment by pradyunsg** [comment]
@brettcannon shouldn't this PR name (and the commit messages) start with "PEP 621:"?

File: pep-0621.rst (hunk 3 above)
Comment (brettcannon) [inline pep-0621.rst:278]:
```suggestion
defined. Other ecosystems have selected "author" as the term to use.
```

**Comment by brettcannon** [comment]
@sinoroc thanks for the catch! Just an FYI, it's a lot easier to accept suggested changes like that when you use [suggestions](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request). That way it's a two button press for me to accept your suggested change.

@pradyunsg I'm a PEP editor and no one else is going to worry about this PR since I'm the author, so prefixing the PR is really not that critical. So sure, but I'm also not worrying about it either. 😄

**Pull request merged**